### PR TITLE
Feature/colse session

### PR DESCRIPTION
**Pull request by mapengbo**
Make sure you have checked _all_ steps below.

### JIRA
- [ ] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2434
    - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a JIRA issue.

### Description
- [ ] Here are some details about my PR, including screenshots of any UI changes:
When processing DAG files, create multiple dagrun, when dagrun is less than max_active_runs。

### Tests
- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
The online test has been done and no problems have been found

### Commits
- [ ] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### Documentation
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
    - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality
- [ ] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by Fokko** [comment]
Please adhere to the guidelines. Open a Jira ticket and clearly explain the problem. Thanks!

**Comment by r39132** [comment]
@mapengbo  Also, please provide some means to test this.

File: airflow/jobs.py [hunk 1]
```diff
@@ -1403,10 +1403,13 @@ def _process_dags(self, dagbag, dags, tis_out):
                 continue
 
             self.log.info("Processing %s", dag.dag_id)
-
+			create_num = 1
```
Comment (skudriashev) [inline airflow/jobs.py:1406]: Bad indent here.

File: airflow/jobs.py [hunk 2]
```diff
@@ -1403,10 +1403,13 @@ def _process_dags(self, dagbag, dags, tis_out):
...
             dag_run = self.create_dag_run(dag)
-            if dag_run:
-                self.log.info("Created %s", dag_run)
+			while create_num < dag.max_active_runs and dag_run:
```
Comment (skudriashev) [inline airflow/jobs.py:1408]: And here.

**Review by skudriashev — CHANGES_REQUESTED** [review]

**Comment by Fokko** [comment]
Please adhere to the commit guidelines.

**Comment by feng-tao** [comment]
the commit name has typo, and need more meaningful name for what this pr is for.

**Comment by r39132** [comment]
I'm closing this. Please reopen when you are ready to respond to feedback!

**Pull request closed without merging**